docs/enterprise/upgrading.mdx

```diff
@@ -6,21 +6,27 @@ sidebar_position: 0
 import Tabs from "@theme/Tabs";
 import TabItem from "@theme/TabItem";
 
-## How to: Upgrade to Weave Gitops Enterprise
+## How to: Upgrade to Weave GitOps Enterprise
 
-:::note gitops >= 0.5.0-rc2 version required
+:::note BEFORE YOU START 
 
-Make sure `gitops version` is reporting a version >= `0.5.0-rc2` before continuing with these instructions. Download releases of Weave Gitops from [the releases page](https://github.com/weaveworks/weave-gitops/releases).
+  Make sure the following software is installed before continuing with these instructions:
+  
+  - `gitops` >= 0.5.0-rc2. Use `gitops version` to check the currently installed version. If needed, download a newer version of Weave GitOps from the [releases page](https://github.com/weaveworks/weave-gitops/releases).
+  - `clusterctl` >= 1.0.1. Use `clusterctl version` to check the currently installed version. If needed, download a newer version of clusterctl from the [releases page](https://github.com/kubernetes-sigs/cluster-api/releases).
```

Comment:
We don't actually require 1.0.1, you should use it for the capd templates we provide... but some people _might_ have to use an older one for compatibility.